### PR TITLE
Index Homosaurus terms to homoit_s

### PR DESCRIPTION
**Pull request by maxkadel**
This vocabulary follows the pattern set by the Siku and local subject headings.

Connected to #1925

### CODE REVIEW

**Comment by maxkadel** [comment]
Is it correct that this is only adding the terms to a single Solr field? Should we be adding them to any of the other subject fields for search?

**Comment by sandbergja** [comment]
@maxkadel that's what the `homoit_subject_unstem_search` should be for.  It will just need to be configured in the [solrconfig](https://github.com/pulibrary/pul_solr/blob/main/solr_configs/catalog-production-v2/conf/solrconfig.xml) as well via a PR to the pul_solr repo.

File: marc_to_solr/lib/traject_config.rb [hunk 1]
```diff
@@ -840,20 +840,27 @@
   accumulator.replace(subjects)
 end
 
-# Adds lc and siku subject unstem_search fields
+# Homosaurus controlled vocabulary
```
Comment (christinach) [inline marc_to_solr/lib/traject_config.rb:843]: @maxkadel great! can you include in the comment the link to the specific vocabulary? https://homosaurus.org/

Comment (maxkadel) [reply marc_to_solr/lib/traject_config.rb]: Done!

**Review by sandbergja — APPROVED** [review]
Thanks @maxkadel

**Pull request merged**